### PR TITLE
SPOI-8411: Added deduper example

### PR DESCRIPTION
**Pull request by bhupeshchawda**
@chinmaykolhatkar / @shubham-pathak22 
Please review and merge

### CODE REVIEW

File: tutorials/dedupExample/src/site/conf/my-app-conf1.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (shubham-pathak22) [inline tutorials/dedupExample/src/site/conf/my-app-conf1.xml]: Is this file needed ?

Comment (bhupeshchawda) [inline tutorials/dedupExample/src/site/conf/my-app-conf1.xml]: Was created by archetype. Removed.

**Comment by bhupeshchawda** [comment]
Updated the PR
